docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -4307,7 +4307,7 @@ Keeping Tables Small
 
 Over time, some of the tables will grow large and this will drag down performance (this is especially true on a wireless setup).
 
-One such table is the `locationlog` table. We recommend that closed entries in this table be moved to the archive table `locationlog_archive` after some time. A closed record is one where the `end_time` field is set to a date (strictly speaking it is when `end_time` is not null and not equals to 0).
+One such table is the `locationlog` table. We recommend that closed entries in this table be moved to the archive table `locationlog_history` after some time. A closed record is one where the `end_time` field is set to a date (strictly speaking it is when `end_time` is not null and not equals to 0).
```

Comment:
This line isn't true anymore, this happens auto-magically so we can just remove this

Reply:
You're right, I missed it. I will adjust.